CDC documentation (#734)
neo4j/docs-cypher: modules/ROOT/pages/clauses/listing-procedures.adoc

|===
| name | description | mode | worksOnSystem

| "cdc.current"

> | "Returns the current change identifier that can be used to stream changes from."

Reviewer: ```suggestion
| "Returns the current change identifier that can be used to stream changes from."
label:new[Introduced in 5.13]
```

| "READ"
| false

| "cdc.earliest"

> | "Returns the earliest change identifier that can be used to stream changes from."

Reviewer: ```suggestion
| "Returns the earliest change identifier that can be used to stream changes from."
label:new[Introduced in 5.13]
```

| "READ"
| false

| "cdc.query"

> | "Query changes happened from the provided change identifier."

Reviewer: I don't understand what "happened from the provided change identifier" means (hence my suggestion may be inaccurate).

```suggestion
| "Query changes observed by the provided change identifier."
label:new[Introduced in 5.13]
```

|===
| name | description | mode | worksOnSystem

> | "cdc.current"
> | "Returns the current change identifier that can be used to stream changes from."
> label:new[Introduced in 5.13]
> | "READ"
> | false
> 
> | "cdc.earliest"
> | "Returns the earliest change identifier that can be used to stream changes from."
> label:new[Introduced in 5.13]
> | "READ"
> | false
> 
> | "cdc.query"
> | "Query changes observed by the provided change identifier."
> label:new[Introduced in 5.13]
> | "READ"
> | false

Reviewer: ```suggestion
| "cdc.current"
| "Returns the current change identifier that can be used to stream changes from."
| "READ"
| false

| "cdc.earliest"
| "Returns the earliest change identifier that can be used to stream changes from."
| "READ"
| false

| "cdc.query"
| "Query changes observed by the provided change identifier."
| "READ"
| false
```
